frontend/src/views/Listen.vue

```diff
@@ -4,13 +4,17 @@
     <div class="pb-3">
       <h3>{{ $title }}</h3>
     </div>
-    <v-card color="#d9edf7">
+    <v-card color="#d9edf7" :max-width="isInternetExplorer ? '90vw' : '100%'">
```

Comment:
Can we use a pure CSS browser targeting instead like suggested here https://stackoverflow.com/a/28419263/3082178?